### ISSUE TITLE
Add the URL to the demo?

### ISSUE THREAD

**Issue by DeckOfPandas** [openhealthcare/fp17-demo#2]
https://openhealthcare.github.io/fp17-demo/

**Comment by lamby** [comment]
Good idea, although 'tis likely this repo will disappear now that @pacharanero is making it part of Opal. Indeed, the static HTML bit was already improved, superceded & bug fixed — including "your" spelling correction — in the Django version :)

**Comment by pacharanero** [comment]
as @lamby says, this repo is likely going to be deprecated in favour of the Opal version, which contains all the excellent datamodelling that was done here
